### PR TITLE
Housekeeping/2488 adapt refactor page header theme to v7 and update design

### LINKED ISSUE #2488: [Housekeeping] xxxAdapt refactor/update page header theme to v7 and update button design
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the housekeeping request

The refactor/update page header theme was originally intended for Kirby v6 without attention-levels #1512 included. It was later decided to postphone the release of this theme until the release of Kirby v7.
The task is to adapt this theme to work with Kirby v7 and update the design to comply with the design changes - talk to @henrikvoetmand

### PR DESCRIPTION
**Pull request by mictro**
## Which issue does this PR close?

This PR closes #2488 

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-16
+>=16.0.0
```
Comment (jkaltoft) [inline .nvmrc:1]: Is this change related to the issue?

Comment (mictro) [reply .nvmrc:1]: No, it was auto-merged like this - work on this issue has started and its perfectly ok that there are several issues that needs to be addressed.
You'll be notified when the issue i ready for inspection.  :)
